examples/dynamics/heavy_hex_ising_real_tebd.jl

```diff
@@ -0,0 +1,190 @@
+using ITensors
+using ITensorNetworks
+using NamedGraphs
+using NamedGraphs: rem_edges!, add_edge!
+using Graphs
+
+using ITensorNetworks:
+  initialize_bond_tensors,
+  vidal_itn_canonicalness,
+  vidal_to_symmetric_gauge,
+  vidal_gauge,
+  approx_network_region,
+  optimal_contraction_sequence,
+  contract,
+  symmetric_to_vidal_gauge,
+  norm_network
+
+#Remove edges from a graph involving the vertex v (either at src or dst)
+function rem_edges_involving_vertex(g::NamedGraph, v)
```

Comment:
Doesn't this happen automatically if the vertex is removed with `rem_vertex!`?

Reply:
You're right it does! Changed this now and removed the function